tests/core/fs/storage/test_user_storage.py

```diff
@@ -100,10 +103,12 @@ async def test_vacuum(alice_user_storage):
 
 @pytest.mark.trio
 @customize_fixtures(real_data_storage=True)
-async def test_storage_file_tree(tmp_path, alice):
+async def test_storage_file_tree(tmp_path: Path, alice: LocalDevice):
     manifest_sqlite_db = tmp_path / alice.slug / "user_data-v1.sqlite"
 
+    # Pristine start: DB is not created on FS...
+    assert not manifest_sqlite_db.exists()
     async with UserStorage.run(tmp_path, alice) as aus:
-        assert aus.manifest_storage.path == manifest_sqlite_db
-
-    assert manifest_sqlite_db.is_file()
+        _ = aus.device
```

Comment:
what is this for ?

Reply:
Because otherwise `aus` isn't used
I've rename `aus` to `_aus`

Reply:
Otherwise `aus` isn't used, now I've renamed it to `_aus`

Reply:
if you don't need aus to you can omit the `as aus` then:
```python
async with UserStorage.run(tmp_path, alice):
    ...
```

Reply:
done